tests/modules/nf-scil/registration/antsapplytransforms/nextflow.config

```diff
@@ -0,0 +1,11 @@
+process {
+
+    withName: "REGISTRATION_ANTSAPPLYTRANSFORMS" {
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+    }
+
+    withName: "REGISTRATION_ANTSAPPLYTRANSFORMS_TEST" {
+        publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+        ext.interpolation = "linear"
+    }
```

Comment:
I'm not sure I see the usefulness of this test. You can just add your task.ext in the first one.